composer.json

```diff
@@ -12,21 +12,22 @@
         }
     },
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^8.0",
         "surfnet/yubikey-api-client": "^2.3",
-        "symfony/config": "^2.7|^3.0|^4.4",
-        "symfony/dependency-injection": "^2.7|^3.0|^4.4",
-        "symfony/http-kernel": "^2.7|^3.0|^4.4"
+        "symfony/config": "^4.4|^5.0|^6.0",
+        "symfony/dependency-injection": "^4.4|^5.0|^6.0",
+        "symfony/http-kernel": "^4.4|^5.0|^6.0",
+        "symfony/yaml": "^6.4"
```

Comment:
Love that you are trying to be defensive here.

Maybe this is a good point to drop SF 4 support? And maybe add provisional SF7 support?

Reply:
Done the first, SF7  we don't know yet